### PR TITLE
fix test generation

### PR DESCRIPTION
**Pull request by GavinJoyce**
/cc @kratiahuja Test generation seems to be incorrect, failing tests were being generated for valid files. I noticed this while working on https://github.com/tildeio/glimmer/issues/243

This fixes the issue

### CODE REVIEW

**Comment by kratiahuja** [comment]
Sorry for the issue Gavin! I'll take a look at it later today. I am away from the laptop right now.

**Comment by GavinJoyce** [comment]
great, thanks!

File: tests/index.js [hunk 1]
```diff
@@ -135,7 +135,24 @@ describe('broccoli-tslinter', function() {
     return builder.build().then(function(results) {
       var dir = results.directory;
       var testGenerated = readFile(dir + '/errorFile1.tslint.ts');
-      assert.equal(testGenerated.indexOf("QUnit.test(\'errorFile1.ts should pass jshint\'"), -1, 'Test should not be generated');
+      assert.equal(testGenerated.indexOf("QUnit.test(\'errorFile1.ts should pass tslint\'"), -1, 'Test should not be generated');
+      assert.notEqual(loggerOutput.length, 0, 'Errors should be seen for linted files');
+    });
+  });
+
+  it('tests should not be generated when disableTestGenerator is false', function() {
```
Comment (kratiahuja) [inline tests/index.js]: The description should be `tests should be generated when disableTestGenerator is false`

**Comment by kratiahuja** [comment]
I think there is one minor issue of the test description, other than this looks good. cc: @GavinJoyce

File: tests/index.js (hunk 1 above)
Comment (GavinJoyce) [inline tests/index.js]: thanks, updated

**Comment by kratiahuja** [comment]
@GavinJoyce 1.0.2 is published!

**Pull request merged**